app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardScreen.kt

```diff
@@ -221,10 +226,26 @@ fun KeyboardScreen(
         Box(
             contentAlignment = alignment,
             modifier = Modifier
+                .conditional(backdropEnabled) {
```

Comment:
Its not necessary to re-create how kotlin does conditionals. Just have a line above this that does

val backdropModifier = if (backdropEnabled) ... else Modifier;

Reply:
I might be misunderstanding what you're suggesting, but this way seemed nicer to me because it's not necessary to have two different modifiers with repeated things, you can just have the one with conditionals for things that aren't necessarily used in all circumstances.